app/lib/submission_builder/base.rb

```diff
@@ -0,0 +1,38 @@
+module SubmissionBuilder
```

Comment:
Shared "Base" class for Submission XML builder classes. some of this logic used to be located in the "Buildable" concern, but as more parts got built out, it became apparent that the builder classes shared a lot of core logic.